### ISSUE TITLE
Import multiple recipes from GDrive

### ISSUE THREAD

**Issue by mcmellow** [caseydavenport/biermacht#82]
Hi,

Is it possible to select more then one recipe (xml) from GDrive and import them all in one go?

Greeting

**Comment by gabeweisz** [comment]
I just tried this and was able to export my whole database from one device and import the whole thing on another. The recipes were all in the same xml file though - you might want to try that.

**Comment by caseydavenport** [comment]
Yeah, that's right - right now you can only select one XML file at a time, but you can have multiple recipes within a single XML file.
